roles/search/molecule/default/molecule.yml

```diff
@@ -15,17 +15,23 @@ platforms:
       - /tmp
     volume_mounts:
       - "/sys/fs/cgroup:/sys/fs/cgroup:ro"
+    groups:
+      - repository
+      - nginx
+      - search
 provisioner:
-  name: ansible
   ansible_args:
-    - --inventory=../../inventory_local.yml
+    - -e
+    - reposearch_shared_secret=dummy
```

Comment:
you forgot to remove host_vars here? is set both here and there or you are testing an edge case when user set both? :D

Reply:
unfortunately I do need both the host_vars and the extra-var. The later is used for converge & idempotency to work has expected. The former is used for feature flip in testinfra tests to work (as it relies on host.ansible.get_variables which only knows about inventory vars not runtime  vars)